google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableInstanceAdminClientIT.java

```diff
@@ -285,6 +285,41 @@ public void basicInstanceOperationTest() {
     assertThat(client.listInstances()).contains(instance);
   }
 
+  @Test
+  public void createInstanceWithAutoscalingTest() {
+    String newInstanceId = prefixGenerator.newPrefix();
+    String newClusterId = newInstanceId + "-c1";
+
+    try {
+      ClusterAutoscalingConfig clusterAutoscalingConfig =
+          ClusterAutoscalingConfig.of(newInstanceId, newClusterId)
+              .setMaxNodes(3)
+              .setMinNodes(1)
+              .setCpuUtilizationTargetPercent(30);
+      client.createInstance(
+          CreateInstanceRequest.of(newInstanceId)
+              .addCluster(
+                  newClusterId,
+                  testEnvRule.env().getPrimaryZone(),
+                  clusterAutoscalingConfig,
+                  StorageType.HDD)
+              .setDisplayName("Multi-Cluster-Instance-Test")
+              .addLabel("state", "readytodelete")
+              .setType(Type.PRODUCTION));
+
+      Cluster cluster = client.getCluster(newInstanceId, newClusterId);
+      assertThat(cluster.getId()).contains(newClusterId);
+      assertThat(cluster.getServeNodes()).isEqualTo(1);
+      assertThat(cluster.getAutoscalingMinServeNodes()).isEqualTo(1);
+      assertThat(cluster.getAutoscalingMaxServeNodes()).isEqualTo(3);
+      assertThat(cluster.getAutoscalingCpuPercentageTarget()).isEqualTo(30);
+    } catch (Exception e) {
+      Assert.fail("error in the test: " + e.getMessage());
```

Comment:
why not let it bubble up?

Reply:
refactored, and updated a few other tests as well